code/modules/jobs/job_types/legion.dm

```diff
@@ -150,84 +150,82 @@ Veteran Decan
 		/obj/item/throwing_star/spear, \
 		/obj/item/claymore/machete/gladius=1, \
 		/obj/item/reagent_containers/pill/patch/healingpowder=2, \
-		/obj/item/storage/box/lethalshot, \
+		/obj/item/ammo_box/m44=2, \
 		/obj/item/flashlight/flare/torch=1, \
-		/obj/item/megaphone/cornu=1, \
 		/obj/item/stack/f13Cash/random/denarius/legionpay_officer)
 
 
 /*
-Decan
+Vexillarius
 */
-/datum/job/CaesarsLegion/Legionnaire/f13decan
-	title = "Decanus"
-	flag = F13DECAN
+/datum/job/CaesarsLegion/Legionnaire/f13vexillarius
+	title = "Legion Vexillarius"
+	flag = F13VEXILLARIUS
 	faction = "Legion"
 	total_positions = 2
 	spawn_positions = 2
 	forbids = "The legion forbids: Ghouls(Anywhere), Women in armed roles, chems and medicine besides healing powder. Lethally harming any servants of Caesar without proper reason."
 	enforces = "The Legion expects: Obeying orders of superiors. A roman style name. Wearing the uniform, unless acting as a nonlethal infiltrator. Expansion of Caesars Domain."
-	description = "You answer directly to the Veteran Decanus and Centurion, working with them as their lieutenants, running the camp and training recruits, gathering intelligence and leading the Centuria on patrols, raids and scouting missions on behalf of the commander."
-	supervisors = "veteran decanus"
+	description = "You answer directly to the Decanus and Centurion, acting as an elite soldier of the Legion you hold the standard of Caesar in battle, raising troop morale and rallying the Centuria to your banner in the heat of battle, guiding them back to you."
+	supervisors = "Decanus and up."
 
 	exp_requirements = 960
 
 
-	outfit = /datum/outfit/job/CaesarsLegion/Legionnaire/f13decan
+	outfit = /datum/outfit/job/CaesarsLegion/Legionnaire/f13vexillarius
 
-/datum/outfit/job/CaesarsLegion/Legionnaire/f13decan
-	name = "Decanus"
-	jobtype = /datum/job/CaesarsLegion/Legionnaire/f13decan
-	shoes = 		/obj/item/clothing/shoes/legionleather
-	suit = 			/obj/item/clothing/suit/armor/f13/legrecruit
-	head = 			/obj/item/clothing/head/helmet/f13/legion/legdecan
-	mask =			/obj/item/clothing/mask/bandana/legdecan
+/datum/outfit/job/CaesarsLegion/Legionnaire/f13vexillarius
+	name = "Vexillarius"
+	jobtype = /datum/job/CaesarsLegion/Legionnaire/f13vexillarius
+	suit = 			/obj/item/clothing/suit/armor/f13/legvexil
+	head = 			/obj/item/clothing/head/helmet/f13/legion/legvexil
 	glasses = 		/obj/item/clothing/glasses/sunglasses/big
-	suit_store = 	/obj/item/gun/ballistic/revolver/m29/alt
-	r_hand = 		/obj/item/twohanded/fireaxe
+	suit_store = 	/obj/item/gun/ballistic/shotgun
 	backpack_contents = list(
-		/obj/item/restraints/legcuffs/bola/tactical=1, \
+		/obj/item/restraints/handcuffs=1, \
+		/obj/item/throwing_star/spear, \
 		/obj/item/throwing_star/spear, \
 		/obj/item/claymore/machete/gladius=1, \
 		/obj/item/reagent_containers/pill/patch/healingpowder=2, \
-		/obj/item/ammo_box/m44=2, \
+		/obj/item/storage/box/lethalshot, \
```

Comment:
They already have boxes and boxes of lethal shot and I'd rather not encourage them to kill each other. Why the change to a shotgun from a revolver in the first place?